### DISCUSSION TITLE
Switching to the Drag tool by holding a key

### DISCUSSION THREAD

**Discussion by korsakovs** [azimuttapp/azimutt#171] [Features]
Hi!

When I m in the "Select tool" mode, it would be nice to be able to temporarily move to the "Drag tool" mode by holding Shift, Alt/Option, or Ctrl/Command key, and getting back once I release the key.

Thank you :)

Regards,
Sergey

**Comment by loicknuchel** [comment]
Hi @korsakovs 

I missed alerts so just saw your message now, sorry for the delay.
This switch between select and drag tool is already implemented with the space bar.
Did you try it?

Have a great day,
Loïc

**Reply by korsakovs** [reply]
Hi @loicknuchel ! Thanks for replying to me! Indeed, it works just fine with the space bar.

**Comment by korsakovs** [comment]
Closing, as the feature was actually implemented.
